### DISCUSSION TITLE
IMDB format changed

### DISCUSSION THREAD

**Discussion by rschellenberg** [Flexget/Flexget#3386] [Q&A] [answered]
Afternoon everyone, Just was looking at logs and noticed this error: because of an error: IMDB parser needs updating, imdb format changed. Please report on Github.

Unfortunately I can't find an easy way to update specific plugins. 
I upgraded the system through pip3, still no joy. 

Any help would be appreciated.

**Comment by stevezau** [comment, marked as answer]
I am addressing it here https://github.com/Flexget/Flexget/pull/3384

**Reply by rschellenberg** [reply]
Thank you Steve for your commitment and effort to this project.
